src/instructlab/eval/mt_bench_common.py

```diff
@@ -272,14 +302,38 @@ def chat_completion_openai(
             )
             output = response.choices[0].message.content
             break
-        except openai.OpenAIError as e:
```

Comment:
Don't we still need to handle this exception as a catchall.  In case they add a new exception type for example.